src/ampycloud/data.py

```diff
@@ -1036,6 +1074,6 @@ def metar_msg(self, which: str = 'layers') -> str:
 
         # Here, deal with the situations when all clouds are above the MSA
         if len(msg) == 0:
```

Comment:
I just realized that we also need to take into account the significance of the layers located between the MSA and MSA+Delta MSA. So the update should look more like something along the following lines:

```
layers_in_msa_buffer = len(sligrolay['significant']*(sligrolay['alt_base'] >= msa_val)) > 0. # Before l. 1072

...

return 'NSC' if layers_in_msa_buffer else self._ncd_or_nsc() . # To replace l. 1077
```

The last line may not be the best, because the code is hardcoded ?

Reply:
I think it's fine to hardcode this as this code will very likely not change. I implemented it along your sugestion.